Jenkinsfile

```diff
@@ -72,7 +72,7 @@ pipeline {
                         rm -f ${SRC_DIR}/pfe/extensions/codewind-odo-extension-*.zip
                         rm -f ${SRC_DIR}/pfe/extensions/odo
                         curl -Lo ${SRC_DIR}/pfe/extensions/codewind-appsody-extension-0.6.0.zip http://download.eclipse.org/codewind/codewind-appsody-extension/master/latest/codewind-appsody-extension-0.6.0.zip
-                        curl -Lo ${SRC_DIR}/pfe/extensions/codewind-odo-extension-0.5.0.zip https://github.com/eclipse/codewind-odo-extension/archive/0.5.0.zip
+                        curl -Lo ${SRC_DIR}/pfe/extensions/codewind-odo-extension-0.6.0.zip https://github.com/eclipse/codewind-odo-extension/archive/0.6.0.zip
```

Comment:
Can we use v0.6.0.zip instead? to avoid a potential collision with the upcoming '0.6.0' branch creation.